encoding/ssz/query/path.go

```diff
@@ -37,24 +59,13 @@ func ParsePath(rawPath string) ([]PathElement, error) {
 	var path []PathElement
```

Comment:
```suggestion
	var elem []PathElement
```
`path` seems quite confusing at this PR. I'm not so sure what's best option: `pathElements`, `elements`, etc.

Reply:
Totally agree with this. Renaming this to `pathElements` because it is the most explicit